message LogoutReq {
	required uint32 nothing = 1;
}
